blockchain/blockchain.go

```diff
@@ -1614,10 +1613,15 @@ func (bc *BlockChain) insertChain(chain types.Blocks) (int, []interface{}, []*ty
 			return i, events, coalescedLogs, err
 		}
 
-		switch status {
+		switch writeResult.Status {
 		case CanonStatTy:
-			logger.Debug("Inserted new block", "number", block.Number(), "hash", block.Hash(),
-				"txs", len(block.Transactions()), "gas", block.GasUsed(), "elapsed", common.PrettyDuration(time.Since(bstart)))
+			processTxsTime := procStats.AfterApplyTxs.Sub(procStats.BeforeApplyTxs)
+			processFinalizeTime := procStats.AfterFinalize.Sub(procStats.AfterApplyTxs)
+			validateTime := afterValidate.Sub(procStats.AfterFinalize)
+			logger.Info("Inserted a new block", "number", block.Number(), "hash", block.Hash(),
+				"txs", len(block.Transactions()), "gas", block.GasUsed(), "elapsed", common.PrettyDuration(time.Since(bstart)),
+				"processTxs", processTxsTime, "processFinalize", processFinalizeTime, "validateState", validateTime,
```

Comment:
to shorten the log a little...
```suggestion
				"processTxs", processTxsTime, "finalize", processFinalizeTime, "validateState", validateTime,
```